InsertFunctions.java

```diff
@@ -0,0 +1,79 @@
+
+import java.sql.*;
+
+
+public class InsertFunctions {
+/*Java class to access and insert new data into the DataBase
+ * for the modules, categories, courses, and streams tables.
+ */
+	
+	
+	 
+	public static void main(String[] args){
+		try {
+			Statement state;
+			Connection c1;
+			Class.forName("oracle.jdbc.driver.OracleDriver");
+			c1 = DriverManager.getConnection("jdbc:oracle:thin:@localhost:1521:XE", "Student_Performance", "Student_Performance");
+			state = c1.createStatement();
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+	}
+	
+	
+		
+		
+		//INSERT FOR STREAM
+		public void insertStream(Statement state, String id, String name){
+		try {
+		int r1 =state.executeUpdate("INSERT INTO stream VALUES('"+id+"', '"+name+"')");

```

Comment:
..... fine. You all win. Let us all pray that Oracle's inbuilt defenses are enough. Because slightly reformatting your query and adding 3 or 4 extra lines is apparently way too much work.